doc/source/dev/roadmap-detailed.rst

```diff
@@ -205,9 +204,10 @@ misc
 ``scipy.misc`` will be removed as a public module.  Most functions in it have
 been moved to another submodule or deprecated.  The few that are left:
 
-- ``info``, ``who`` : these are NumPy functions
 - ``derivative``, ``central_diff_weight`` : remove, possibly replacing them
   with more extensive functionality for numerical differentiation.
+- ``ascent``, ``face``, ``electrocardiogram`` : remove or move to the
+  appropriate subpackages (e.g. ``scipy.ndimage``, ``scipy.signal``).
```

Comment:
`info` and `who` do not seem to be public parts of `misc` anymore. The others are publicly documented.
https://docs.scipy.org/doc/scipy/reference/misc.html